alpine-consul/root/etc/cont-init.d/00-consul-template

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/with-contenv sh
+
+# add s6 specific down file to tell not to start service
+if [ ! -e "/etc/services.d/consul-template/down" ]; then
```

Comment:
Just a note to myself really, referencing `/etc/services.d` is fine at this point, as s6-overlay stage 2 hasn't copied them over to `/var/run/s6/services` (which happens as the very next step).